source/MRMesh/MRMeshLoadObj.cpp

```diff
@@ -121,15 +120,11 @@ tl::expected<std::vector<NamedMesh>, std::string> fromSceneObjFile( std::istream
             std::string str;
             std::getline( in, str );
         }
-        if ( callback )
+        if ( callback && !(i % 1000) )
```

Comment:
Worth looking into assembly, probably (i % 1024) == (i & 1023) is much faster to compute. Here and in all other places. Probably create an inline function to avoid copy-paste.

Reply:
fixed

Reply:
in this particular place?